Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -106,60 +107,85 @@ class MeasurementSelector {
       return CombinatorialKalmanFilterError::MeasurementSelectionFailed;
     }
 
+    assert(!cuts->chi2CutOff.empty());
+    const auto& chi2CutOff = cuts->chi2CutOff;
+    auto maxChi2Cut = *std::max_element(chi2CutOff.begin(), chi2CutOff.end());
```

Comment:
I wonder if we should add a global maximum to the config. that way we would not need to look for the maximum all the time

Reply:
would make sense to me.